frontend/src/components/submission/SubmissionForm.vue

```diff
@@ -247,6 +261,25 @@ onBeforeMount(async () => {
         help-text="Optionally provide a number between -114 and -139"
         :disabled="!props.editable"
       />
+      <InputText
+        class="col-4"
+        name="streetAddress"
+        label="Location address"
+        :disabled="!props.editable"
+      />
+      <InputText
+        class="col-4"
+        name="singleFamilyUnits"
+        label="Units"
+        :disabled="!props.editable"
+      />
+      <Dropdown
+        class="col-4"
+        name="isRentalUnit"
+        label="Rental units"
+        :disabled="!props.editable"
+        :options="RentalStatusList"
```

Comment:
oh. i just looked at Tyler's instructions.. i guess Unsure is a value they want.
but the field should be empty by default where currently i think it will default to 'no'.. which could be a problem

Reply:
maybe the RentalStatusList constant just needs re-ordering so 'Unsure' is first in the array

Reply:
Would default to 'Unsure' in this case. The field also needs to be added to the form schema validation as a oneOf.